test/test_torch.py

```diff
@@ -363,6 +364,16 @@ def test_remainder(self):
             res2[i, 3] = res2[i, 3] % q
         self.assertEqual(res1, res2)
 
+        # Check the LongTensor case
```

Comment:
The tests in [`test_cuda.py`](https://github.com/pytorch/pytorch/blob/master/test/test_cuda.py#L202) verify that the resuts between the CPU implementation and the GPU implementation are the same, no? (Although
 the CI build hasn't caught the bug I had in THCTensorMathPairwise.cu, I wonder why)
I'll add the tests for negative divisors.